Extension/test/scenarios/SimpleCppProject/tests/languageServer.integration.test.ts

```diff
@@ -85,8 +87,10 @@ suite("multiline comment setting tests", function(): void {
 /* **************************************************************************** */
 
 function cppPropertiesPath(): string {
-    const wf = vscode.workspace.workspaceFolders?.[0] ?? assert.fail("No workspace folder open");
-    return `${wf.uri.fsPath}/.vscode/c_cpp_properties.json`;
+    const folder = vscode.workspace.workspaceFolders?.[0];
+    assert.ok(folder, 'workspace folder snould be set.');
```

Comment:
`snould`->`should` (there are 4 other cases below.

Reply:
This wasn't fixed?

Reply:
it is now